db/migrate/20170528155838_create_users.rb

```diff
@@ -1,4 +1,6 @@
 class CreateUsers < ActiveRecord::Migration[5.1]
+  acts_as_voter
```

Comment:
Does this belong in here? (I don't know, but it feels weird to have it in a migration)